README.md

```diff
@@ -238,23 +238,22 @@ While your _Working Directory_ and _Staging Area_ don't really care about branch
 
 You can think of _branches_ in git as pointers, pointing to a series of commits. When you `commit`, you add to whatever you're currently pointing to. 
 
-Just adding a branch, doesn't directly take you there, it just creates such a pointer. 
+Just adding a branch, doesn't directly take you there, it just creates such pointer. 
```

Comment:
I'm not a native English speaker, so please correct me, but I feel the _a_ is fine there? 
Why exactly did you rewrite this?

Reply:
Neither am I a native speaker, but it felt to me that _a_ was redundant here.
I've walked through some grammar references and you seem to be right. Will revert this change.